lib/more_collections.nit

```diff
@@ -123,6 +137,20 @@ class HashMap3[K1, K2, K3, V]
 		end
 		level2[k2, k3] = v
 	end
+
+	# Remove the item at `k1`, `k2` and `k3`
+	fun remove_at(k1: K1, k2: K2, k3: K3)
```

Comment:
Unrelated: Took me so much time to establish the link with `HashMap[23]` in the comment commit :P